src/Microsoft.ML.Transforms/NAReplaceTransform.cs

```diff
@@ -18,31 +18,32 @@
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.Runtime.Model;
 using Microsoft.ML.Runtime.Model.Onnx;
+using Microsoft.ML.Core.Data;
```

Comment:
>using Microsoft.ML.Core.Data [](start = 0, length = 28)

Why VS can't sort it properly, why, oh why?

Reply:
after re-activating resharper, the sorting problem went away. 

---
In reply to: [217855717](https://github.com/dotnet/machinelearning/pull/917#discussion_r217855717) [](ancestors = 217855717)

Reply:
Let's do what Ctrl-R-G does (which is alphabetically)

---
In reply to: [217875183](https://github.com/dotnet/machinelearning/pull/917#discussion_r217875183) [](ancestors = 217875183,217855717)